superset/assets/javascripts/SqlLab/components/AceEditorWrapper.jsx

```diff
@@ -8,6 +8,22 @@ import { areArraysShallowEqual } from '../../reduxUtils';
 
 const langTools = ace.acequire('ace/ext/language_tools');
 
+const keywords = (
```

Comment:
what's the advantage of storing these as a string rather than an array of keywords?